web_ui/frontend/app/registry/namespace/components/NamespaceForm.tsx

```diff
@@ -0,0 +1,174 @@
+                    </Select>
+                    <FormHelperText>Read Only: Caches are declared with a &apos;/cache&apos; prefix</FormHelperText>
+                </FormControl>
+            </Box>
+            <Box pb={2}>
+                <TextField
+                    fullWidth
+                    size={"small"}
+                    id={"pubkey"}
+                    name={"pubkey"}
+                    label={"Pubkey"}
+                    variant={"outlined"}
+                    multiline={true}
+                    defaultValue={namespace?.pubkey || ""}
+                    inputProps={{
+                        style: {
+                            fontFamily: "monospace",
+                            fontSize: "0.8rem",
+                            lineHeight: "0.9rem",
+                            minHeight: "1.5rem"
+                        }
+                    }}
+                />
+            </Box>
+            <Box pb={2}>
+                <TextField
+                    fullWidth
+                    size={"small"}
+                    id={"description"}
+                    name={"description"}
+                    label={"Description"}
+                    variant={"outlined"}
+                    defaultValue={namespace?.admin_metadata?.description || ""}
+                />
+            </Box>
+            <Box pb={2}>
+                <TextField
+                    fullWidth
+                    size={"small"}
+                    id={"site-name"}
+                    name={"site-name"}
+                    label={"Site Name"}
+                    variant={"outlined"}
+                    defaultValue={namespace?.admin_metadata?.site_name || ""}
+                />
+            </Box>
+            <Box pb={2}>
+                <FormControl fullWidth size={"small"}>
+                    <InputLabel id="institution-label">Institution</InputLabel>
+                    <Select
+                        labelId="institution-label"
+                        id="institution"
+                        name={"institution"}
+                        label="Institution"
+                        value={institution}
+                        onChange={event => setInstitution(event.target.value as string)}
+                    >
+                        {institutions.map(institution => <MenuItem key={institution.id}
+                                                                   value={institution.id}>{institution.name}</MenuItem>)}
+                    </Select>
+                </FormControl>
+            </Box>
+            <Box pb={2}>
+                <TextField
+                    fullWidth
+                    size={"small"}
+                    id={"Security Contact ID"}
+                    name={"security-contact-user-id"}
+                    label={"Security Contact ID"}
+                    variant={"outlined"}
+                    defaultValue={namespace?.admin_metadata?.security_contact_user_id || ""}
```

Comment:
Also a help text here: "CILogon User Identifier of the user responsible for the security of the service"

Reply:
Fixed